waku/v2/node/wakunode2.nim

```diff
@@ -406,14 +395,14 @@ proc mountFilter*(node: WakuNode) =
 
 # NOTE: If using the swap protocol, it must be mounted before store. This is
 # because store is using a reference to the swap protocol.
-proc mountSwap*(node: WakuNode, swapConfig: SwapConfig = SwapConfig.init()) =
+proc mountSwap*(node: WakuNode, swapConfig: SwapConfig = SwapConfig.init()) {.raises: [Defect, LPError].} =
```

Comment:
There is already a `{.push raises: [Defect].}` at the top, do we need another one in here?

Reply:
The top-level gets _replaced_ by the function specific raises `{.raises: [Defect, LPError].}`. https://status-im.github.io/nim-style-guide/errors.exceptions.html. `{.raises: [Defect]}` is too strict for this function which raises an `LPError` too, hence it needs its own annotation.